src/test/java/uk/gov/justice/probation/courtcaseservice/controller/OffenderController_ConvictionIntTest.java

```diff
@@ -38,7 +40,7 @@
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT, properties = "org.apache.catalina.connector.RECYCLE_FACADES=true")
 public class OffenderController_ConvictionIntTest {
```

Comment:
In setUp, it's setting featureFlags.setFlagValue("fetch-attendance-data",true) - is that used ?